newa/__init__.py

```diff
@@ -407,12 +408,18 @@ def get_errata(self, event: Event) -> list[Erratum]:
             for item in builds_json:
                 builds += list(item.keys())
             if builds:
-                errata.append(Erratum(id=event.id,
-                                      respin_count=int(info_json["respin_count"]),
-                                      summary=info_json["synopsis"],
-                                      people_assigned_to=info_json["people"]["assigned_to"],
-                                      release=release,
-                                      builds=builds))
+                errata.append(
+                    Erratum(
+                        id=event.id,
+                        content_type=ErratumContentType(
+                            info_json["content_types"][0]),
```

Comment:
Can we a check that there are some content_types? Or do you think it is impossible that at some point that key can have empty list?

Reply:
TBH I have no idea if there can be erratum without a content type. But what about something like this?
```
if not info_json["content_types"]:
    warn('Erratum XX does not have a content type)
    content_type = None
else:
    content_type = ErratumContentType(info_json["content_types"][0])
...
...
Erratum(
                        id=event.id,
                        content_type=content_type
...

```

Reply:
It is just was nitpick, info_json["people"]["assigned_to"] is not handled too so maybe we can just leave it as it is (if there is no list there it will raise an Exception anyway).

Reply:
OK. Let's leave it as it is then. :-)